client/src/components/ExportCsv.tsx

```diff
@@ -1,46 +1,90 @@
-import { ExportToCsv } from "export-to-csv"
+import {formatIsoDate} from 'date/utils';
+import {ExportToCsv} from 'export-to-csv';
 
-function csvExporter(user: any) {
-    const data = [
-        {
-            displayName: user.displayName,
-            approvals: user.aggregatedAuthoredTotals.approvals,
-            rejections: user.aggregatedAuthoredTotals.rejections,
-            totalComments: user.aggregatedAuthoredTotals.commentsWrittenTotal,
-            commentsToOthers: user.aggregatedAuthoredTotals.commentsWrittenToOthers,
-            changeRequests: user.aggregatedAuthoredTotals.changesCreated,
-            commits: user.aggregatedAuthoredTotals.commits,
-            averageTimeInReview: user.aggregatedAuthoredTotals.meanChangeOpenTimeMsec,
-            commentsPerRequest: user.aggregatedAuthoredTotals.commentsWrittenToOthers / user.aggregatedReceivedTotals.reviewRequests,
-          },
-        // "test",
-    ];
-    const options = { 
-        fieldSeparator: ',',
-        quoteStrings: '"',
-        decimalSeparator: '.',
-        showLabels: true, 
-        showTitle: true,
-        title: 'User Code Review Stats',
-        filename: `${user.displayName}-stats.csv`,
-        useTextFile: false,
-        useBom: true,
-        useKeysAsHeaders: true,
-      };
-    const csvExporter = new ExportToCsv(options);
-    csvExporter.generateCsv(data);
-}
-
-const ExportCsv = ( {user} : any ) => {
+const sum = (acc: number, item: number) => acc + item;
 
-    return (
-        <div className="text-right">
-            <button className="btn btn-primary" onClick={() => csvExporter(user)}>
-                Export to CSV
-            </button>
-        </div>
-    )
+function csvExporter(users: any) {
+  // const data = [];
+  let data = users.map((row: any) => {
```

Comment:
this can probably be `const`

EDIT: I think you're fixing this in a subsequent commit. That's fine, but it'd be better if each commit is atomic and clean from the start. If individual commits are broken, using tools like `git bisect` is a lot harder, as well as reasoning about why a feature was a built in a particular way.

You could use `git rebase` to squash the commits together.